.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: MIT App Inventor Community
+    url: https://community.appinventor.mit.edu/
```

Comment:
I would have it point specifically to https://community.appinventor.mit.edu/c/mit-app-inventor-help/5

Reply:
Fixed, it now points to that URL